src/components/heading/ApartsFilter.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {useContext} from 'react';
+import {Context} from "../../Context";
+import MySelectApart from "./MySelectApart";
+
+
+
+function ApartsFilter() {
+    
+  const {currentApart, setCurrentApart} = useContext(Context);
+
```

Comment:
We definitely need to have a linter in this project, meanwhile use "Format document" feature in code editor you're working in